src/lib/header-bidding/prebid/bid-config.spec.ts

```diff
@@ -145,26 +142,11 @@ describe('getImprovePlacementId', () => {
 			[createAdSize(1, 2)],
 		];
 
-		return prebidSizes.map((size) => getImprovePlacementId(size, false));
+		return prebidSizes.map((size) => getImprovePlacementId(size));
 	};
 
 	test('should return -1 if no cases match', () => {
-		expect(getImprovePlacementId([createAdSize(1, 2)], false)).toBe(-1);
-	});
-
-	test('should give the expected values when in the fronts banner test in uk on desktop', () => {
-		isInUk.mockReturnValue(true);
-		getBreakpointKey.mockReturnValue('D');
-		containsMpuOrDmpu.mockReturnValue(false);
-		containsLeaderboardOrBillboard.mockReturnValueOnce(true);
-		containsBillboardNotLeaderboard.mockReturnValue(true);
```

Comment:
I think it would be good to still have a test to cover the containsBillboardNotLeaderboard case for Improve - with these changes we don't test for the 22987847 return value any more

Reply:
It would be good (but a proper chore!) to improve the readability of these tests at some point.

Reply:
Completely agree! I'm very reliant on searching for specific values to navigate these tests